doc/rfc-move-sigverify-out-of-mask-rom.md

```diff
@@ -54,6 +54,7 @@ The immutable section would not be included in the signature for the ROM extensi
 
 1. The mask ROM containing sigverify has already been taped-out and tested. Changing it would reduce the utility of existing test results.
 2. Integrated flash is more expensive per-byte than mask ROM in terms of area/silicon cost.
+   * The size of sigverify is approximately 10Kib as of Earlgrey 1.0.0.
```

Comment:
Nit: `10 Kib` is ten kilobits (e.g. 1.25 kilobytes).  I think you mean `10 KiB`.

Reply:
Yes I do, good catch